plenum/test/client/test_protocol_version.py

```diff
@@ -13,6 +15,24 @@ def request_num(request):
     return int(request.param)
 
 
+def test_client_send_incorrect_ledger_status(looper, txnPoolNodeSet,
+                                             sdk_pool_handle,
+                                             sdk_wallet_client):
+    # SDK client send LEDGER_STATUS without protocoloVersion.
+    # Node discard this message and send client's LEDGER_STATUS back
+    sdk_send_random_and_check(looper, txnPoolNodeSet,
```

Comment:
SDK will send LedgerStatus with protocol version in the next version. So, this test will fail.

Reply:
fixed